src/App.tsx

```diff
@@ -15,35 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  BY_LENGTH = 'length',
+  ALPHABETICALLY = 'alphabetically',
+  DEFAULT = '',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [sortOrder, setSortOrder] = useState<boolean>(true);
+
+  function prepareGoods(goods: string[], sortBy: string, order = true) {
```

Comment:
The function's name `prepareGoods` does not start with a verb and is not descriptive of the action it performs. Consider renaming it to more accurately reflect its functionality, for example `sortGoods` or `getSortedGoods` to align with the [CODE STYLE] guidelines.